test/lib/common_test.js

```diff
@@ -83,6 +83,26 @@ exports.kubernetesFilesGenerator = (applicationName) => [
   exports.kubernetesDeploymentFileGenerator(applicationName),
   exports.kubernetesServiceFileGenerator(applicationName)
 ]
+exports.jenkinsfile = 'Jenkinsfile'
+exports.debianInstall = 'debian/install'
+exports.debianControl = 'debian/control'
+exports.terraformStart = 'terraform/scripts/start.sh'
+exports.terraformVariables = 'terraform/variables.tf'
+exports.vsiFiles = [ 'debian/changelog',
+  'debian/compat',
+  'debian/rules',
+  'terraform/main.tf',
+  'terraform/output.tf',
+  'terraform/scripts/build.sh',
+  'terraform/scripts/fetch-state.sh',
+  'terraform/scripts/install.sh',
+  'terraform/scripts/publish-state.sh',
+  exports.jenkinsfile,
```

Comment:
the Jenkinsfile is not VSI specific (nor does VSI rely on a Jenkinsfile). I'm not certain if it is necessary to include checks on the Jenkinsfile here